bangank36/zoomify-generator
# 🗺️ Zoomify Generator: Implement preview tiles with PhotoSwipe

## Summary
- Using photoswipe plugin to preview the generated images, the selected format is zoomify (plugin should dzi format too)
- Note that current script in `run.php` did not produce a proper `full.jpg` in the output folder

## Testing
- Open localhost at localhost:8000
- Make sure the image is loaded properly
![chrome-capture-2023-2-19](https://user-images.githubusercontent.com/10071857/226173972-692fd6e6-5b75-4182-b5a3-af3fc4130e38.gif)


## Reference
#4
